libs/langgraph/src/pregel/remote.ts

```diff
@@ -105,16 +105,12 @@ const getStreamModes = (
     (typeof streamMode === "string" ||
       (Array.isArray(streamMode) && streamMode.length > 0))
   ) {
-    if (typeof streamMode === "string") {
-      updatedStreamModes.push(streamMode);
-    } else {
-      reqSingle = false;
-      updatedStreamModes.push(...streamMode);
-    }
+    reqSingle = typeof streamMode === "string";
+    const mapped = Array.isArray(streamMode) ? streamMode : [streamMode];
+    updatedStreamModes.push(...mapped);
```

Comment:
This is equivalent to what was there before right?

Reply:
Yep!